compiler/circle-tensordump/src/Dump.cpp

```diff
@@ -95,7 +95,7 @@ void print_buffer(std::ostream &os, uint32_t buff_idx, const flatbuffers::Vector
   os << std::endl;
 }
 
-} // namepsace
+} // namespace
```

Comment:
This fix is not a bug-fix but a typo-fix but let's go as-is

Reply:
Thank you :) I did not want to include this, but this was automatically included by the format checker.